src/animation/AnimationMixer.js

```diff
@@ -753,6 +753,16 @@ AnimationMixer.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 		}
 
+	},
+
+	dispose: function () {
+
+		EventDispatcher.prototype.dispose.call( this );
+
+		this.stopAllAction();
+
+		// TODO is that it?
```

Comment:
(This comment to be removed) Is that all there is todo with this class? Never used it before.